internal/enginecache/cache.go

```diff
@@ -214,11 +214,13 @@ func ValidateOrUpdateEngineAtPath(targetOS string, cachePath string) (engineCach
 		os.Exit(1)
 	}
 	cachedEngineVersion := string(cachedEngineVersionBytes)
-	requiredEngineVersion := FlutterRequiredEngineVersion()
+	if len(requiredEngineVersion) == 0 {
+		requiredEngineVersion = FlutterRequiredEngineVersion()
+	}
 
 	if cachedEngineVersion != "" {
 		if cachedEngineVersion == requiredEngineVersion {
-			log.Printf("Using engine from cache")
+			log.Printf("Using engine from cache: %s/%s", cachedEngineVersionPath, cachedEngineVersion)
```

Comment:
Maybe this is to verbose? The normal user doesn't need to know the used engine version and a user who overwrote the engine version does know which engine version they want to use. I'd keep it like before.

Reply:
otherwise LGTM

Reply:
you okay with putting as a debug log? i had to go digging to find this stuff =)

Reply:
Yep